Command/StartCrawlerCommand.php

```diff
@@ -47,13 +48,25 @@ public function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+	$blacklist = [];
+	
+	if(count($input->getOption('blacklist')) > 0){	    
```

Comment:
I am wondering, what is the added value of outputting this information? You already specified the blacklist rules as options.